core/Azure.Mcp.Core/tests/Azure.Mcp.Tests/Client/CommandTestsBase.cs

```diff
@@ -16,27 +18,57 @@ public abstract class CommandTestsBase(ITestOutputHelper output) : IAsyncLifetim
     protected const string TenantNameReason = "Service principals cannot use TenantName for lookup";
 
     protected McpClient Client { get; private set; } = default!;
-    protected LiveTestSettings Settings { get; private set; } = default!;
+    public LiveTestSettings Settings { get; set; } = default!;
```

Comment:
I wouldn't expect anything other than a derived class would need access to the test class's Setting property